packages/cra-template-typescript/template/README.md

```diff
@@ -6,23 +8,23 @@ In the project directory, you can run:
 
 ### `npm start`
 
-Runs the app in the development mode.<br />
+Runs the app in the development mode.\
```

Comment:
[MD033 - Inline HTML](https://github.com/DavidAnson/markdownlint/blob/v0.20.4/doc/Rules.md#md033)

Reply:
I didn't realize you could use `\` to create a line break.